src/js/themes/base.d.ts

```diff
@@ -874,7 +887,10 @@ export interface ThemeType {
     size?: string;
     font?: {
       weight?: number | string;
-    }
+    };
+  };
+  radioButtonGroup?: {
+    gap?: GapType;
```

Comment:
This should accommodate your recent changes.

Reply:
gotcha, just to clarify, something like this:

```
radioButtonGroup?: {
  a11yTitle: string;
  align: string;
  alignContent: string;
  ...
  ...
}
```

Reply:
Similar to: 
```
  card?: {
    container?: BoxProps;
  },
```